### PR TITLE
improve:update Xr.h/XRCommon.h

### PR DESCRIPTION
**Pull request by oahc09**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by oahc09** [comment]
@cocos-robot run test cases

**Comment by oahc09** [comment]
@cocos-robot run test cases

**Comment by oahc09** [comment]
@cocos-robot run test cases

File: native/cocos/platform/interfaces/modules/XRCommon.h [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
+     i.  Bypass or avoid any relevant technical protection measures in the products or services;
+     ii. Release the source codes to any other parties;
+     iii.Disassemble, decompile, decipher, attack, emulate, exploit or reverse-engineer these portion of code;
+     iv. Apply it to any third-party products or services without Licensor��s permission;
+     v.  Publish, copy, rent, lease, sell, export, import, distribute or lend any products containing these portions of code;
+     vi. Allow others to use any services relevant to the technology of these codes; and
+     vii.Conduct any other act beyond the scope of this Licensing Agreement.
+ g.  This Licensing Agreement terminates immediately if the Licensee breaches this Agreement. The Licensor may claim compensation from the Licensee where the Licensee��s breach causes any damage to the Licensor.
+ h.  The laws of the People's Republic of China apply to this Licensing Agreement.
+ i.  This Agreement is made in both Chinese and English, and the Chinese version shall prevail the event of conflict.
```
Comment (minggo) [inline native/cocos/platform/interfaces/modules/XRCommon.h]: Why change the license?

Comment (oahc09) [reply native/cocos/platform/interfaces/modules/XRCommon.h]: @QingliWang

**Review by minggo — APPROVED** [review]

**Pull request merged**